recipes/crispritz/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set sha256 = "10a98877a892a66ca39aadb4ab9590c52c427aaa29bd03735a5a6215ee406ac4" %}
+
+package:
+  name: crispritz
+  version: 1.0
+
+source:
+  url: https://github.com/InfOmics/CRISPRitz/archive/master.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - {{compiler('cxx')}} =5.4
```

Comment:
Please get rid of `=5.4`, that's likely to break other packages in the same env.